kwave/executor.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, device):
         if self._is_linux:
             binary_folder = 'linux'
         elif self._is_windows:
-            binary_folder = 'windows'
+            binary_folder = 'cygwin'
```

Comment:
Do we support `cygwin` only? If we support different systems in Windows, I would suggest keeping the name `windows` to be more general